pipeline_api/src/test/java/com/splicemachine/pipeline/client/BulkWriteActionTest.java

```diff
@@ -61,9 +60,7 @@
 
     @Before
     public void setUp() throws Exception{
-        PipelineTestDataEnv pipelineTestDataEnv =PipelineTestEnvironment.loadTestDataEnvironment();
-
-        this.pef = pipelineTestDataEnv.pipelineExceptionFactory();
```

Comment:
Same here, we don't need this change, this has to run using the classpath from `hbase_pipeline`

Reply:
removed this commit